src/libraries/Common/src/System/Net/Http/aspnetcore/NetEventSource.Common.cs

```diff
@@ -472,17 +473,17 @@ private static string Format(FormattableString s)
             switch (s.ArgumentCount)
             {
                 case 0: return s.Format;
-                case 1: return string.Format(s.Format, Format(s.GetArgument(0)));
-                case 2: return string.Format(s.Format, Format(s.GetArgument(0)), Format(s.GetArgument(1)));
-                case 3: return string.Format(s.Format, Format(s.GetArgument(0)), Format(s.GetArgument(1)), Format(s.GetArgument(2)));
+                case 1: return string.Format(CultureInfo.CurrentCulture, s.Format, Format(s.GetArgument(0)));
```

Comment:
FWIW, this actually hurts perf rather than helps (though in this context of logging it's not going to make a big difference).  The only reason to get the current culture here is if the particular argument being formatted needs it.  Let's say the argument is an Int32: if no culture was provided and if the number is positive, the implementation won't actually access the current culture, because nothing about the culture will impact how it's formatted.  So this change means that we're now accessing the TLS for CurrentCulture when we may not have to.